modules/proc_splunk-config.adoc

```diff
@@ -0,0 +1,66 @@
+:_content-type: PROCEDURE
+[id="proc_splunk-config"]
+= Configuring {productname} to use Splunk
+
+Use the following procedure to configure {productname} to use Splunk.
+
+.Prerequisites
+
+* You have installed Splunk and created a username.
+* You have generated a Splunk bearer token.
+
+.Procedure
+
+. Open your {productname} `config.yaml` file and add the following configuration fields:
++
+[source,yaml]
+----
+---
+LOGS_MODEL: splunk
+LOGS_MODEL_CONFIG:
+    producer: splunk
+    splunk_config:
+        host: http://<user_name>.remote.csb <1>
+        port: 8089 <2>
+        bearer_token: <bearer_token> <3>
+        url_scheme: <http/https> <4>
+        verify_ssl: False <5>
+        index_prefix: <splunk_log_index_name> <6>
+        ssl_ca_path: <location_to_ssl-ca-cert.pem> <7>
+---
+----
+<1> String. The Splunk cluster endpoint.
+<2> Integer. The Splunk management cluster endpoint port. Differs from the Splunk GUI hosted port. Can be found on the Splunk UI under *Settings* -> *Server Settings* -> *General Settings*.
+<3> String. The generated bearer token for Splunk.
+<4> String. The URL scheme for access the Splunk service. If Splunk is configured to use TLS/SSL, this must be `https`.
+<5> Boolean. Whether to enable TLS/SSL. Defaults to `true`.
+<6> String. The Splunk index prefix. Can be a new, or used, index. Can be created from the Splunk UI.
+<7> String. The relative container path to a single `.pem` file containing a certificate authority (CA) for TLS/SSL validation. Must be a full chain certificate, for example, a complete SSL/TLS certificate chain that includes the server certificate, intermediate certificate(s), and root certificate.
```

Comment:
Based on the testing results, when only provide the rootCA Cert, it works well.

Reply:
@LiZhang19817 fixed.